src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpResponseStream.Managed.cs

```diff
@@ -86,7 +87,9 @@ private void DisposeCore()
                     // Ignore error due to connection reset by peer
                 }
             }
+#pragma warning disable CA1416 // Validate platform compatibility
             _response.Close();
```

Comment:
Probably `DisposeCore()` should have had [UnsupportedOSPlatform("browser")], but that causing more warnings suppressing for now

Reply:
Why is _response.Close();` not available on browser, but all the other usages of `_response` available?

---
In reply to: [504146716](https://github.com/dotnet/runtime/pull/43363#discussion_r504146716) [](ancestors = 504146716)

Reply:
I would like to know the answer too, somehow it was annotated with Unsupported browser https://github.com/dotnet/runtime/pull/42063/files#diff-855eb66c05b5f1c31df24ecdcc7d26cc4f9b26f928c22c68bfc5163036255f22R96 cc @MaximLipnin

Reply:
`M:System.Net.HttpListenerResponse.Close` was marked among other APIs in https://github.com/dotnet/runtime/pull/42063 because it was recognized by PNSE scanner as throwing on browser.

Reply:
Well the main question was why only `_response.Close();` is unsupported but all others not, anyway seems we decided to mark the entire assembly unsupported on browser with https://github.com/dotnet/runtime/pull/43401, so these suppressions not needed anymore